r2/kv/store.go

```diff
@@ -109,7 +109,8 @@ func (s *store) CreateNamespace(namespaceID string, uOpts r2.UpdateOptions) (*ru
 		return nil, r2.NewNotFoundError(fmt.Sprintf("namespace: %s does not exist", namespaceID))
 	}
 
-	view, err := ns.NamespaceView(0)
+	// Get the latest view of the namespace.
+	view, err := ns.NamespaceView(len(ns.Snapshots()) - 1)
```

Comment:
I think we might need to clean up the views at some point, right now some views are sorted by time in descending order, others are sorted by time in ascending order, which is confusing.